tasks/task3.c

```diff
@@ -0,0 +1,64 @@
+#include <stdio.h>
+#include <stdlib>
+#include "task3.h"
+
+typedef struct Node{
+    int value;
+    struct Node* next;
+}Node;
+
+Node* createNode(int value)
+{
+    Node* newNode = (Node*)malloc(sizeof(Node));
+    if(newNode == NULL){
+        exit(1)
+    }
+    newNode -> value = value;
+    newNode -> next = NULL;
+    return newNode;
+}
+
+void push(Node** head, int data)
+{
+    Node* newNode = (Node*)malloc(sizeof(Node));
+    newNode -> value = data;
+    newNode -> next = (*head);
```

Comment:
Так писать очень не принято. Обычно
```suggestion
    newNode->value = data;
    newNode->next = (*head);
```